environments/jvm/pom.xml

```diff
@@ -21,7 +21,7 @@
 		</dependency>
 		<dependency>
 			<groupId>io.fission</groupId>
-			<artifactId>fission-java-core</artifactId>
+			<artifactId>fission-java-spring</artifactId>
```

Comment:
In order to support backward compatibility - we should let `fission-java-core` be as it is